### ISSUE TITLE
Distro?

### ISSUE THREAD

**Issue by Dexxa1024** [jeffmvr/mangle-vpn#3]
What distro does this run on, Ive tried it on Centos 7 but cant seem to get it to install

**Comment by jeffmvr** [comment]
Yes this should work under CentOS 7. What is the specific error you receive?

**Comment by Dexxa1024** [comment]
[root@hostpanel1 ~]# sudo git clone https://github.com/jeffmvr/mangle-vpn.git /opt/mangle-vpn
Cloning into '/opt/mangle-vpn'...
remote: Enumerating objects: 1074, done.
remote: Total 1074 (delta 0), reused 0 (delta 0), pack-reused 1074
Receiving objects: 100% (1074/1074), 1.10 MiB | 0 bytes/s, done.
Resolving deltas: 100% (646/646), done.
[root@hostpanel1 ~]# cd /opt/mangle-vpn
[root@hostpanel1 mangle-vpn]# sudo ./install.sh
Failed to execute operation: No such file or directory
[root@hostpanel1 mangle-vpn]#

Thats what it says on centos,  It installed on ubuntu, but cant connect any websites as it says no internet which is odd,
